config/initializers/cookie_store.rb

```diff
@@ -10,7 +10,6 @@
                                          secure: true,
                                          redis: {
                                            expire_after: 1.day,
-                                           ttl: 1.day,
```

Comment:
`ttl` is no longer a valid arg, not much documentation on an alternative but I can keep looking if we need to keep this?

Reply:
try `expire_after` instead of `ttl` ?

Reply:
ignore me, I see expire_after is already set